modules/apps/commerce/commerce-product-test/src/testFunctional/tests/SFCommerceOrders.testcase

```diff
@@ -585,6 +585,86 @@ definition {
 		}
 	}
 
+	@description = "This is a use case for COMMERCE-3111. Assert that user can see an open order after deleting the product, and the product is no longer in the order."
+	@priority = "3"
+	test CanSeeAnOpenOrderWithADeletedProduct {
```

Comment:
The test name does not reflect the actual case. Since the test completes the checkout, so the order turns into PLACED, could you please rename it into canSeeAPlacedOrderWithDeletedProduct? Thanks.